mirage/config.ts

```diff
@@ -89,6 +89,7 @@ export default function(this: Server) {
         only: ['index'],
         path: '/files/:parentID/versions',
     });
+    osfNestedResource(this, 'file', 'cedarMetadataRecords', { only: ['index'] });
```

Comment:
Not sure if this is going to be changed in a later PR, but I think this  `{ only: ['index'] }` is going to be a problem when adding/updating cedarMetadataRecords later.

Reply:
Nevermind, the create and updates are going to the top-level endpoint, and not this nested endpoint!